backtrack/src/app/blacklist/blacklist.page.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, NgZone } from '@angular/core';
```

Comment:
make sure you add the info() method even if its empty- otherwise it won't compile!